airflow/configuration.py

```diff
@@ -414,6 +420,7 @@ def run_command(command):
 job_heartbeat_sec = 1
 scheduler_heartbeat_sec = 5
 authenticate = true
+max_threads = 2
```

Comment:
I was thinking "threads" is better understood as a concept. The executor uses ```parallelism``` but that is also a bit unclear.